src/main/webapp/components/Table.js

```diff
@@ -149,7 +158,29 @@ export const Table = hh(class Table extends Component {
               return <TableHeaderColumn key={header.name}
                 dataField={header.value}
                 dataFormat={this.formatStatusColumn}
-                dataSort={true}>{header.name}</TableHeaderColumn>
+                dataSort={true}
+                width={'140px'}>{header.name}</TableHeaderColumn>
```

Comment:
These hard-coded values should be in a configurable style object elsewhere. Something like
```
const styles = { 
  statusWidth: 140,
  fileTypeWidth: 170,
  etc.
}
```
and then you can refer to it directly, `styles.statusWidth`, etc.

Reply:
Also, pretty sure you don't need the px in the string - I think just using the raw number will default to the pixels.